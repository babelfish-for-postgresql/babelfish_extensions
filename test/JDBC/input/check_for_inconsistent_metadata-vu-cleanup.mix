DROP FUNCTION IF EXISTS check_for_inconsistent_metadata_vu_prepare_func
GO
