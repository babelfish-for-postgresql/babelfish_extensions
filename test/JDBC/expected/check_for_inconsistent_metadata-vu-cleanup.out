DROP VIEW IF EXISTS check_for_inconsistent_metadata_vu_prepare_view
GO

DROP PROCEDURE IF EXISTS check_for_inconsistent_metadata_vu_prepare_proc
GO

DROP FUNCTION IF EXISTS check_for_inconsistent_metadata_vu_prepare_func
GO

DROP TABLE IF EXISTS check_for_inconsistent_metadata_table
GO
